apps/wallet/src/ui/app/shared/text/index.tsx

```diff
@@ -30,13 +30,15 @@ const textStyles = cva([], {
             'gray-75': 'text-gray-75',
             'gray-70': 'text-gray-70',
             'gray-65': 'text-gray-65',
+            'gray-60': 'text-gray-60',
             'sui-dark': 'text-sui-dark',
             sui: 'text-sui',
             'sui-light': 'text-sui-light',
             steel: 'text-steel',
             'steel-dark': 'text-steel-dark',
             'steel-darker': 'text-steel-darker',
             'hero-dark': 'text-hero-dark',
+            'success-dark': 'text-success-dark',
```

Comment:
Can you also add these changes to Text over in the explorer so that these components are in-sync?